cli/src/lib.rs

```diff
@@ -463,6 +465,14 @@ pub enum ClusterCommand {
 }
 
 pub fn entry(opts: Opts) -> Result<()> {
+    let toolchain_config = override_toolchain(&opts.cfg_override)?;
+    let result = process_command(opts);
+    restore_toolchain(&toolchain_config)?;
```

Comment:
Killing the process before that would leave the environment switched. Is there a way we can connect everything only for this runtime?